src/simulator/privacy_tracker.py

```diff
@@ -89,7 +112,26 @@ def privacy_consumption(self) -> PrivacyBudget:
         basic composition rule.  This will be expanded in the future
         to support advanced composition (see TODO #1 above).
```

Comment:
Minor comment about previous PR...  where is TODO #1?  Will it make sense to move this comment to the `privacy_consumption_for_sampling_bucket` method below

Reply:
TODO #1 is under PrivacyTracker. I've added the comment also in privacy_consumption_for_sampling_bucket so that we don't forget to remove there; but I'm also keep this comment here for clarity.